requirements.txt

```diff
@@ -10,7 +10,7 @@
 daemonize==2.4.7; sys.platform != 'win32'
 
 # Install Shotgun API 3 from archive
-https://github.com/shotgunsoftware/python-api/archive/v3.5.1.zip
+https://github.com/shotgunsoftware/python-api/archive/v3.6.2.zip
```

Comment:
I always wonder which "format" is better between this one and:
```
git+https://github.com/shotgunsoftware/python-fernet.git@v1.0.1.1
```

Reply:
I'd personally prefer the `git+https` format. We can do a full scan to rewrite it like that everywhere in the future.